packages/form-js-editor/rollup.config.js

```diff
@@ -70,7 +70,7 @@ export default [
         targets: [
           { src: 'assets/form-js-editor-base.css', dest: 'dist/assets' },
           { src: '../../node_modules/@bpmn-io/draggle/dist/dragula.css', dest: 'dist/assets' },
-          { src: '../../node_modules/@bpmn-io/properties-panel/assets/properties-panel.css', dest: 'dist/assets' }
+          { src: '../../node_modules/@bpmn-io/properties-panel/dist/assets/properties-panel.css', dest: 'dist/assets' }
```

Comment:
Why is this necessary?

Reply:
There was a change in the core properties panel that now also stores the styles inside `dist` (for consistency reasons). It's not a breaking change as the old path is still there, but I found it better to follow the best practice sooner than later.

Reply:
https://github.com/bpmn-io/properties-panel/blob/main/CHANGELOG.md#340